plugins/org.jkiss.dbeaver.ext.db2/src/org/jkiss/dbeaver/ext/db2/DB2SQLDialect.java

```diff
@@ -106,6 +108,15 @@ protected void loadFunctions(JDBCSession session, JDBCDatabaseMetaData metaData,
         }
     }
 
+    @NotNull
+    @Override
+    protected String getProcedureCallEndClause(DBSProcedure procedure) {
+        if (procedure.getProcedureType() == DBSProcedureType.FUNCTION) {
+            return "FROM SYSIBM.SYSDUMMY1";
```

Comment:
Probably you should leave a comment with a link to the source this goofy table comes from

https://www.ibm.com/docs/en/db2-for-zos/12?topic=tables-sysdummy1